app/assets/javascripts/covid19/src/webapp/components/structures-table/IDRDialog.tsx

```diff
@@ -0,0 +1,210 @@
+import _ from "lodash";
+import React from "react";
+import styled from "styled-components";
+import { Typography } from "@material-ui/core";
+import { Pagination } from "@material-ui/lab";
+import { ListItemProps } from "./cells/DetailsCell";
+import { colors } from "./badge/Badge";
+import { Dialog } from "./Dialog";
+import { IDROptions } from "./Columns";
+import { Assay, Compound, Screen } from "../../../domain/entities/LigandImageData";
+import i18n from "../../../utils/i18n";
+
+export interface IDRDialogProps {
+    onClose(): void;
+    idrOptions?: IDROptions;
```

Comment:
`flag?: Type` is ok for optional things and you don't want the bother the user to pass a value (as it has a default). Here, it's more like: `idrOptions: Maybe<IDROptions>;`.

Reply:
I don't understand so well.